gui/default/index.html

```diff
@@ -122,8 +122,8 @@ <h3 class="panel-title">
               <li ng-if="authenticated"><a href="" ng-click="advanced()"><span class="fa fa-fw fa-cogs"></span>&nbsp;<span translate>Advanced</span></a></li>
               <li ng-if="authenticated"><a href="" ng-click="logging.show()"><span class="fa fa-fw fa-wrench"></span>&nbsp;<span translate>Logs</span></a></li>
 
-              <li class="divider" aria-hidden="true" ng-if="config.gui.debugging"></li>
-              <li><a href="/rest/debug/support" target="_blank" ng-if="config.gui.debugging"><span class="fa fa-fw fa-user-md"></span>&nbsp;<span translate>Support Bundle</span></a></li>
+              <li ng-if="authenticated" class="divider" aria-hidden="true"></li>
```

Comment:
How about removing this divider? I was considering filing a PR doing that separately, but the reason is that "Logs" and "Support Bundle" are kind of related to each other.

Alternatively, the divider could be moved up, so that "Advanced" is separated from "Logs" and "Support Bundle".

Reply:
Agreed... it verges on being worth a separate PR, but perhaps grouping settings and advanced and moving show ID (which is easily accessible from the main screen by tapping the local ID), so

```
Settings
Advanced
---
Logs
Support bundle
Show ID
---
Log out
Restart
Shut down
```
```

Reply:
How about in this order?

```
Settings
Advanced
---
Show ID
Logs
Support bundle
---
Log out
Restart
Shut down
```

I think the first three are probably much more used than the rest 🙂.

> show ID (which is easily accessible from the main screen by tapping the local ID)

This is kind of true, however if your screen is narrow, "This Device" needs a lot of scrolling to get to, so using "Actions" → "Show ID" is much faster.